iib/workers/tasks/build.py

```diff
@@ -1220,6 +1227,7 @@ def handle_rm_request(
                 binary_image=prebuild_info['binary_image'],
                 from_index=from_index_resolved,
                 overwrite_from_index_token=overwrite_from_index_token,
+                container_tool='podman',
```

Comment:
I wonder whether do we want to have this hardcoded as it is or configurable so it can be turned on/off without need to rewrite the code.

Did someone discuss about this?

Reply:
The reason I did not make it configurable is because it is not used for every request. It is only used for non-fbc builds which are all EOL as of today and this piece of code will only cease to exist in the months to come. Having said that, I am open to making it configurable if guild members think otherwise.

Reply:
Got it, thanks!